pkg/smartcontract/param_type.go

```diff
@@ -139,6 +141,35 @@ func (pt *ParamType) DecodeBinary(r *io.BinReader) {
 	*pt = ParamType(r.ReadB())
 }
 
+// EncodeDefaultValue writes a script to push the default parameter value onto
+// the evaluation stack into the given writer. It's mostly useful for constructing
+// dummy invocation scripts when paramet types are known, but they can't be
+// filled in. A best effort approach is used, it can't be perfect since for many
+// types the exact values can be arbitrarily long, but it tries to do something
+// reasonable in each case. For signatures, strings, arrays and "any" type a 64-byte
+// zero-filled value is used, hash160 and hash256 use appropriately sized values,
+// public key is represented by 33-byte value while 32 bytes are used for integer
+// and a simple push+convert is used for boolean. Other types produce no code at all.
+func (pt ParamType) EncodeDefaultValue(w *io.BinWriter) {
+	var b [64]byte
+
+	switch pt {
+	case AnyType, SignatureType, StringType, ByteArrayType:
```

Comment:
Are you sure we'd like to do this for Any, strings and byte arrays? It's pretty clear for signature, but other types are questionable.

Reply:
Well, we either ignore them or try doing something like this. None of the approaches is perfect, but maybe the way it is now it'll handle a bit more cases (the most likely thing to be accepted by `verify` is some kind of signature).